openai/version.py

```diff
@@ -1 +1 @@
-VERSION = "0.7.0"
+VERSION = "0.7.1"
```

Comment:
Even though we're still a `0.x`, I tend to prefer using minor, instead of patch, as the version bump for new features. So `0.8.0`